IndicoV2.Abstractions/IndicoV2.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We should move to the latest LTS some time next quarter for all of these 🤔... not for this PR, just so maybe one of us remembers :)